source/integrations-guide/integrations-guide-index.rst

```diff
@@ -6,370 +6,127 @@ Integrations Guide
    :hidden:
    :titlesonly:
 
-   popular-integrations
-   run-slash-commands
+   plugins
    webhook-integrations
-
-Mattermost provides a variety of methods to integrate with your favorite tools, automate critical workflows, and extend the capabilities of the platform. This guide provides a high-level overview of integration options and the level of technical skills required :ref:`(no-code, low-code, or pro-code) <integrations-guide/integrations-guide-index:what is meant by no-code, low-code, and pro-code?>`, and links to detailed documentation for each.
-
-Connect your tools
-------------------
-
-Out of the box, you can connect with many popular tools without writing any code, and for deeper customization you can extend Mattermost with slash commands, webhooks, bots and API access.
+   slash-commands
+   restful-api
+   no-code-automation
+   faq
+
+Mattermost provides a variety of methods to integrate with your favorite tools, automate critical workflows, and extend the capabilities of the platform. This guide provides a high-level overview of integration options and the level of technical skills required :ref:`(no-code, low-code, or pro-code) <integrations-guide/faq:what is meant by no-code, low-code, and pro-code?>`, and links to detailed documentation for each.
+
+Choose Your Path
+-----------------
```

Comment:
This table is a bit hard to read. Wonder if it would make sense to put each item under "best options" on a new line, then in the examples column each line corresponds to the associated example of that option.

ie 

```
no code | Pre-built plugins                   | Connect Jira, GitHub, Zoom
        | No code automation                  | Automate with Zapier